src/services/ApiService.ts

```diff
@@ -158,6 +160,40 @@ export class ApiService {
             } catch (e) {
                 return false;
             }
+        } else if (ApiService.RuntimeConfig.oidcClientId) {
+            try {
+                if (!this.oidcAxiosInstance) {
+                    //console.log("Creating OIDC Axios instance")
```

Comment:
Please remove comment.

I wonder what the changes in `package-lock.json` are for?